Doc/reference/compound_stmts.rst

```diff
@@ -771,10 +771,8 @@ Coroutine function definition
    keyword: await
 
 Execution of Python coroutines can be suspended and resumed at many points
-(see :term:`coroutine`).  Inside the body of a coroutine function, ``await`` and
-``async`` identifiers become reserved keywords; :keyword:`await` expressions,
-:keyword:`async for` and :keyword:`async with` can only be used in
-coroutine function bodies.
+(see :term:`coroutine`). :keyword:`await` expressions, :keyword:`async for` and
+:keyword:`async with` can only be used in the body of a coroutine function.
```

Comment:
I think it would be worth adding a `versionchanged` clause which indicates when the behaviour changed and what the previous behaviour was.

Reply:
I think you are right, I'll add it.

Reply:
Done.